server.js

```diff
@@ -1,46 +1,46 @@
-var express = require('express');
-var fs = require('fs');
-var request = require('request');
-var cheerio = require('cheerio');
-var app = express();
+const express = require('express');
+const fs = require('fs');
+const request = require('request');
+const cheerio = require('cheerio');
+const app = express();
+
+const URL = 'http://www.imdb.com/title/tt5052448/?ref_=fn_al_tt_1'
+const TITLE = '.title_wrapper > h1[itemprop="name"]'
```

Comment:
I like the use of variables here, so that, in the future, your code can more easily be repurposed to scrape other sites.